spec/acceptance/sequence_context_spec.rb

```diff
@@ -22,7 +22,7 @@ def awesome
   it 'invokes the correct method on the instance' do
     FactoryBot.define do
       factory :sequence_with_public_method, class: User do
-        sequence(:id) { |n| public_method(:awesome).call }
+        sequence(:id) { |_n| public_method(:awesome).call }
```

Comment:
I think when the block doesn't use the yielded value, it doesn't need to even account for it. This should work the same:

```ruby
sequence(:id) { public_method(:awesome).call }
```

Reply:
You're absolutely right. Thanks for pointing that out.